src/zrc/api/validators.py

```diff
@@ -204,3 +213,126 @@ class ZaakBesluitValidator:
     def __call__(self, zaak: Zaak):
         if zaak.zaakbesluit_set.exists():
             raise serializers.ValidationError(self.message, code=self.code)
+
+
+class EitherFieldRequiredValidator:
+    default_message = _("One of %(fields)s must be provided")
+    default_code = "invalid"
+
+    def __init__(self, fields: Iterable[str], message: str = "", code: str = ""):
+        self.fields = fields
+        self.message = (message or self.default_message) % {"fields": ", ".join(fields)}
+        self.code = code or self.default_code
+
+    def __call__(self, attrs: dict):
+        values = [attrs.get(field, None) for field in self.fields]
+        if not any(values):
+            raise serializers.ValidationError(self.message, code=self.code)
+
+
+class JQExpressionValidator:
+    message = _("This is not a valid jq expression.")
+    code = "invalid"
+
+    def __call__(self, value: str):
+        try:
+            jq.compile(value)
+        except ValueError:
+            raise serializers.ValidationError(self.message, code=self.code)
+
+
+class ObjectTypeOverigeDefinitieValidator:
+    code = "invalid"
+
+    def __call__(self, attrs: dict):
+        object_type_overige_definitie: Optional[dict] = attrs.get(
+            "object_type_overige_definitie"
+        )
+        object_url: str = attrs.get("object", "")
+
+        if not object_type_overige_definitie:
+            return
+
+        # object type overige definitie can only be used with object URL reference
+        if object_type_overige_definitie and not object_url:
```

Comment:
```suggestion
        if not object_url:
```

Reply:
As I understood this is only required when the type of object is 'overige'. So the original line (if object_type_overige and not object_url) is the proper validator.